pkg/eth/helper.go

```diff
@@ -59,23 +59,29 @@ func NewETHClientHelper(ethAPIURL string, accountKeys map[string]string) (*Helpe
 }
 
 // NewSimulatedBackendHelper creates a new Helper using an ethereum SimulatedBackend
+// generates accounts for "genesis", "alice", "bob", "carol", "dan", "erin"
 func NewSimulatedBackendHelper() (*Helper, error) {
-	key, err := crypto.GenerateKey()
-	if err != nil {
-		return nil, err
-	}
-	auth := bind.NewKeyedTransactor(key)
 	alloc := make(core.GenesisAlloc)
-	alloc[auth.From] = core.GenesisAccount{Balance: big.NewInt(1000000000)}
-	blockchain := backends.NewSimulatedBackend(alloc, 1000000000)
 
 	accounts := make(map[string]Account)
-	accounts["genesis"] = Account{Key: key, Auth: auth, Address: GetEthAddressFromPrivateKey(key)}
+
+	accountNames := []string{"genesis", "alice", "bob", "carol", "dan", "eric"}
+
+	for _, name := range accountNames {
+		account, err := MakeAccount()
+		if err != nil {
+			return nil, err
+		}
+		accounts[name] = account
+		alloc[account.Address] = core.GenesisAccount{Balance: big.NewInt(9223372036854775807)}
```

Comment:
is that value `922...` random or has some meaning to it?  maybe a const?

Reply:
no significance, I think I found it in another part of the code